### ISSUE TITLE
[Bug] Unable to login to the Minecraft Launcher after installing the Overwolf FTB App 

### ISSUE THREAD

**Issue by SnoopTheMiner** [FTBTeam/FTB-App-Overwolf#21]
EDIT: I'm on Windows 10.

**App Version:**
App: 202012021741-eaf41def11-debugging

**UI Version:**
UI: 9e6200dd

**Log Files:**
**REQUIRED**
 https://pste.ch/zosobahiju

**Describe the bug**
Right after I installed the new FTB App with Overwolf and installed a modpack, I became unable to log into my Mojang account through the Minecraft Launcher through both the launcher opened from a modpack and the vanilla Minecraft launcher.

For some reason, the Minecraft Launcher now thinks that I am offline, and I can no longer connect to the Minecraft services through the launcher. I am still able to log into my account through the Mojang website, so I know that my account has not been compromised. 

My default installation for 1.16.4 on the Minecraft Launcher also appeared to have been modified somehow, because it now displays this message: https://i.imgur.com/AEOVnhn.png (it now is trying to launch the default profile with Optifine, but it did not display this before installing the FTB App).
 
**To Reproduce**
Steps to reproduce the behaviour:
1. Have Overwolf and Minecraft already installed on your computer
2. Install the latest Windows FTB App from the FTB website: https://www.feed-the-beast.com/
3. Install a modpack (I installed FTB Interactions).
4. Try to launch the pack; Minecraft now cannot login or connect to the mojang servers at all, inside or outside of the FTB App.

**Expected behaviour**
I should be able to log into the Minecraft Launcher when I have internet. I'm able to log into other games and go to websites.

**Additional context**
In the FTB App, I've noticed that the button below settings is constantly loading: https://i.imgur.com/pYtOtfT.png
I attempted to make an account through the built in MineTogether system, but the link to the website did not work initially, and now it is just loading. The Friend's List button only makes this loading screen pop up: https://i.imgur.com/ExpPhpn.png

I could not find a solution to this on the internet; the only thing I found was to check/clear my hosts file, but there was nothing in there. I have not used a third party launcher on this computer before.

**Comment by Solverbolt** [comment]
I have the exact same issue

**Comment by SnoopTheMiner** [comment]
> I have the exact same issue

Have you figured out a solution yet? I have already tried the following:

- disabling firewall
- disabling antivirus
- turning off/disabling a vpn
- flushing my DNS (I'm using Google's 8.8.8.8) and doing a winsock reset
- restarting my computer/minecraft/internet

I might try reinstalling Minecraft/ftb next. Looking in the Minecraft Launcher logs, I don't notice anything immediately telling, it just doesn't think I'm connected to the internet.

I DID notice that when I attempted to ping authserver.mojang.com, it didn't return anything. _The ping failed_, which is concerning since I'm able to go to the website in my browser and it still pops up, so it seems like it's something more complicated.

Somehow, it started working again briefly (for about a day) before happening again, so I have no idea what this could be.
